### PR TITLE
Update repo structure

### PR DESCRIPTION
**Pull request by sven1103**

### CODE REVIEW

**Review by KochTobi — APPROVED** [review]
LGTM

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Sven Fillinger
```
Comment (jenniferboedker) [inline LICENSE]: should be university of tübingen

**Comment by jenniferboedker** [comment]
Also the new workflow files are missing (release, publish-snapshots)

**Review by jenniferboedker — CHANGES_REQUESTED** [review]
See my previous comments

**Review by jenniferboedker — APPROVED** [review]
LGTM

**Pull request merged**